vc_zoom/indico_vc_zoom/task.py

```diff
@@ -0,0 +1,41 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2002 - 2021 CERN
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+import datetime
+
+from indico.core import signals
+from indico.core.celery import celery
+from vc_zoom.indico_vc_zoom.api import ZoomIndicoClient
+from vc_zoom.indico_vc_zoom.util import ZoomMeetingType
+
+
+@signals.event.updated.connect
+@signals.event.contribution_updated.connect
+@signals.event.session_block_updated.connect
+def check_meetings(target, changes):
+    if 'start_dt' not in changes:
+        return
+    prev_date, new_date = changes.get('start_dt')
+    if new_date < prev_date:
+        return
+    refresh_meetings.delay(target.vc_room_associations, target.start_dt)
+
+
+@celery.task(name='reschedule_meeting', plugin='vc_zoom')
```

Comment:
IIRC we usually do not specify a task name in plugins but rather keep the auto-generated one